.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
```

Comment:
I think we're all-in on GitHub Actions at this point, and moving away from Circle.

https://renaissancelearning.atlassian.net/wiki/spaces/EN/pages/1442513923/Open+Source+Maintenance#Migrate-to-GitHub-Actions

For what it's worth, I've stopped worrying about having a "nightly" Job and coverage support, mostly to ease the transition. Sorry if I directed you at a since-inapplicable example at some point :grimacing: